root/inc/release-tools.html

```diff
@@ -64,13 +70,20 @@
     <li><% position %></li>
     <%- END %>
 </ul>
-<%- END %>
+<%- END;
+stable = [];
+FOREACH version IN versions;
+    IF version.maturity == 'released';
+        stable.push(version);
+        NEXT;
+    END;
+END %>
```

Comment:
This might be better off as a check to see if there's a release with `release.status == 'latest'` and then short-circuit at that point.  If there's no release with a `latest` status, then the `Latest version` link won't work.

Reply:
Good point. I'll make that adjustment. Thanks.

Reply:
I've updated the pull request with that change.

Reply:
https://github.com/metacpan/metacpan-web/blob/377054534562a2989d1cc48dd125b35a9273644b/root/inc/release-tools.html#L73-L80